goss-testing/scripts/k8s_kyverno_pods_running.sh

```diff
@@ -41,7 +41,9 @@ done
 
 # Checks if all kyverno pods are running. There should be a total of 3 running pods.
 
-running_pods=$(kubectl get poda -n kyverno -o json | jq '[.items[] | select(.metadata.labels.app == "kyverno").status.containerStatuses[0].state.running] | length')
+
+running_pods=$(kubectl get pods -n kyverno -o json | jq '[.items[] | select(.metadata.labels.app == "kyverno").status.containerStatuses[0].state.running] | length')
```

Comment:
It looks like this isn't actually changing anything anymore, the PR has been "updated" from GitHub using the "merge" function (vs. the rebase function". Now the diff seems to show removing and then adding the same line but with surrounding new lines.

Seems this PR is now a no-op.

Reply:
Will check and log a new PR for the above change since this PR is already merged. The change is "poda"  to "pods". 
Thank you.

Reply:
I will close this pull request since mitch has logged one and merged the change.